addons/web/static/src/legacy/js/fields/relational_fields.js

```diff
@@ -1169,10 +1169,11 @@ var FieldX2Many = AbstractField.extend(WidgetAdapterMixin, {
         const _super = this._super.bind(this);
         if (this.view) {
             this._renderButtons();
-            this._controlPanelWrapper = new ComponentWrapper(this, ControlPanelX2Many, {
-                cp_content: { $buttons: this.$buttons },
-                pager: this.pagingState,
-            });
+            const controlPanelContext = this._getControlPanelContext();
+            this._controlPanelWrapper = new ComponentWrapper(this,
+                this._getControlPanelComponent(),
+                controlPanelContext
+            );
```

Comment:
As your function is well named, I would remove the intermediate const.

```suggestion
            this._controlPanelWrapper = new ComponentWrapper(this,
                this._getControlPanelComponent(),
                this._getControlPanelContext()
            );
```

Reply:
True